devicon.json

```diff
@@ -2158,6 +2158,27 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "hardhat",
+	"altnames": [
+		"hardhat-ide"
+	], 
+        "tags": [
+            "ide",
+            "editor",
+            "development-environment"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#fef04c",
+        "aliases": []
```

Comment:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
        ]
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient